nimbus.fml.yaml

```diff
@@ -215,13 +215,20 @@ types:
       description: The identifiers for the sections of the MR 2022.
       variants:
         feature-ui:
-          description: Indicates if the user interfaces should be visible.
+          description: An integer either 0 or 1 indicating if the UI for cookie banner handling should be visible,
+          0 to hide the UI and 1 to show the UI. The actual UI is composed by cookie banner section 
+          in the settings page, the toolbar section and the re-engagement dialog.
         feature-setting-value:
-          description: Indicates if the cookie handling setting should be enabled.
+          description: An integer either 0 or 1 indicating if cookie banner setting should be enabled or disabled,
+            0 for setting the value to disabled, 1  for enabling the setting with the value reject_all.
         dialog-re-engage-time:
-          description: Indicates after how many hours of interaction, the dialog should show again.
+          description: An integer indicating the number of hours that needs to happen before
+          the re-engagement dialog shows again since the last seen, for example if set to 4 
+          that means if the users has seen the dialog, it will see it 4 hours later.
```

Comment:
This probably also needs an indent?

Reply:
I re-push adding indentation thanks 🙏🏽 !